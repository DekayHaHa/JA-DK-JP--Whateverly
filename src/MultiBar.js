import React, { Component } from 'react';
import './MultiBar.css';

export default class SearchBar extends Component {
  constructor() {
    super()
    this.state = {}
  }
  handleToggle = event => {
    event.preventDefault();
  }
  render() {
    return (
      <div className="multi-bar search-section">
        <form>
          <button className="button" onClick={this.handleToggle}> MULTIPLAYER</button>
        </form>
      </div>
    )
  }
}