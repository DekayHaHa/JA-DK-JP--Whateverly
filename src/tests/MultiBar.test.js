import React from 'react';
import ReactDOM from 'react-dom';
import MultiBar from '../MultiBar';
import { shallow } from 'enzyme';

describe('MultiBar', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(
      <MultiBar />
    );
  });
  it('should match snapshop', () => {
    expect(wrapper).toMatchSnapshot();
  });
})