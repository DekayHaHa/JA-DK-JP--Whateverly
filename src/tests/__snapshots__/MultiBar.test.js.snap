// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MultiBar should match snapshop 1`] = `
<div
  className="multi-bar search-section"
>
  <form>
    <button
      className="button"
      onClick={[Function]}
    >
       MULTIPLAYER
    </button>
  </form>
</div>
`;
